lib/presentation/pages/dashboard/widgets/site_item.dart

```diff
@@ -95,12 +95,19 @@ class SiteItem extends StatelessWidget {
                   const SizedBox(
                     width: 5,
                   ),
-                  Text(
-                    site.address ?? "",
-                    style: TextStyle(
-                        color: context.colors.primary,
-                        fontSize: 12,
-                        fontWeight: FontWeight.w600),
+                  Expanded(
+                    child: Text(
+                      site.address ?? "",
+                      maxLines: 2,
+                      overflow: TextOverflow.ellipsis,
+                      style: TextStyle(
```

Comment:
use context.typographies.<text_style> to easily edit the text style for the entire app